Whack-a-mole/index.html

```diff
@@ -0,0 +1,38 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Whack A Mole!</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <h1>Whack-a-mole!<span class="score">0</span></h1>
+    <h2>Record!<span class="record">0</span></h2>
```

Comment:
we need to update `records` if we have the value in a local storage

Reply:
Please improve label as "Record: X" where X is 0 or the value from the local storage

Reply:
done